src/test/java/pageObjects/DynamicLoading.java

```diff
@@ -6,6 +6,7 @@
 public class DynamicLoading extends Base {
 
     int test = 0;
+    String anothertest = "test";
```

Comment:
Not a good idea to include this here since this String variable will never be used. Remove it please.